image.go

```diff
@@ -469,6 +469,14 @@ func (c *Client) ImportImage(opts ImportImageOptions) error {
 	return c.createImage(&opts, nil, opts.InputStream, opts.OutputStream, opts.RawJSONStream, opts.InactivityTimeout, opts.Context)
 }
 
+// BuilderVersion represents either the BuildKit or V1 ("classic") builder.
+type BuilderVersion string
+
+const (
+	BuilderV1       BuilderVersion = "1"
```

Comment:
should this be `BuilderClassic`?

Reply:
The client SDK ([here](https://github.com/moby/moby/blob/5f369ff1d4c8cc2162f865d11f2fbcd5d612013d/api/types/client.go#L221)) calls it BuilderV1, but I've no real preference.  Should I change it?

Reply:
Let's keep it V1 then. Thanks for checking!